src/components/sbb-expansion-panel-content/sbb-expansion-panel-content.scss

```diff
@@ -0,0 +1,21 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-expansion-panel-content-padding-inline-start: var(--sbb-spacing-fixed-6x);
+}
+
+:host([data-icon-space]) {
+  --sbb-expansion-panel-content-padding-inline-start: calc(
+    var(--sbb-spacing-fixed-6x) + var(--sbb-spacing-fixed-9x) + var(--sbb-spacing-fixed-4x)
+  );
+}
+
+.sbb-expansion-panel-content {
+  padding-block-end: var(--sbb-spacing-responsive-xs);
+  padding-inline-start: var(--sbb-expansion-panel-content-padding-inline-start);
+  padding-inline-end: var(--sbb-spacing-fixed-6x);
```

Comment:
should probably be a var, as it is used in data-icon-space definitions as well?

Reply:
Fixed in commit a9406aff